integration_tests/gimbal.cpp

```diff
@@ -82,13 +84,91 @@ TEST_F(SitlTest, GimbalTakeoffAndMove)
     std::this_thread::sleep_for(std::chrono::seconds(3));
 }
 
+TEST_F(SitlTest, GimbalROIOffboard)
+{
+    DroneCore dc;
+
+    ConnectionResult ret = dc.add_udp_connection();
+    ASSERT_EQ(ret, ConnectionResult::SUCCESS);
+
+    // Wait for device to connect via heartbeat.
+    std::this_thread::sleep_for(std::chrono::seconds(2));
+
+    Device &device = dc.device();
+    auto telemetry = std::make_shared<Telemetry>(device);
+    auto gimbal = std::make_shared<Gimbal>(device);
+    auto action = std::make_shared<Action>(device);
+    auto offboard = std::make_shared<Offboard>(device);
+
+    while (!telemetry->health_all_ok()) {
+        LogInfo() << "waiting for device to be ready";
+        std::this_thread::sleep_for(std::chrono::seconds(1));
+    }
+
+    const Telemetry::Position &position = telemetry->position();
+
+    // set the ROI location: a bit to the north of the vehicle's location
+    const double latitude_offset_deg = 3. / 111111.; // this is about 3 m
```

Comment:
Funny, why is it `111111`?

Reply:
I got this from here: https://gis.stackexchange.com/questions/2951/algorithm-for-offsetting-a-latitude-longitude-by-some-amount-of-meters. I tested it and it looked reasonable.